nifi-nar-bundles/nifi-poi-bundle/nifi-poi-services/src/main/java/org/apache/nifi/excel/ExcelReader.java

```diff
@@ -83,6 +112,25 @@ public class ExcelReader extends SchemaRegistryService implements RecordReaderFa
             .addValidator(StandardValidators.POSITIVE_INTEGER_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor PROTECTED_STRATEGY = new PropertyDescriptor
+            .Builder().name("Protected Strategy")
+            .displayName("Protected Strategy")
```

Comment:
The wording of `Protected Strategy` does not seem as an intuitive as it could be when considering some of the [Microsoft documentation on protection and security](https://support.microsoft.com/en-us/office/protection-and-security-in-excel-be0b34db-8cb6-44dd-a673-0b3e3475ac2d).

From one angle, this property controls how the Reader will attempt to process the spreadsheet, so it could be thought of an the `Decryption Strategy`. From the perspective of the spreadsheet, this might make more sense as `Protection Type`, being either `Unprotected` or `Password Protected`. What do you think about `Protection Type` as the name for this property and the associated `enum`?

Reply:
Sounds good to me. I will make those changes. Thanks!